test/env.spec.js

```diff
@@ -27,13 +30,37 @@ describe('env', function () {
     }
   })
 
+  it('isElectronMain should have the correct value in each env', function () {
+    if (env.isElectron && !env.isElectronRenderer) {
+      return expect(env.isElectronMain).to.be.true()
+    }
+    if (env.isElectronMain) {
+      return expect(env.isElectronMain).to.be.true()
```

Comment:
This only asserts that `env.isElectronMain` is true when it is true (or truthy, at least).  Perhaps it should assert that `env.isElectron` is true and `env.isElectronRenderer` is false when `env.isElectronMain` is true?

It also needs to assert that we actually hit some these of assertions when the test is run.

Reply:
added the throw at the end, this way that code makes more sense.